app/hotels/src/map/MapView.js

```diff
@@ -4,7 +4,7 @@ import idx from 'idx';
 import * as React from 'react';
 import { createFragmentContainer, graphql } from 'react-relay';
 import { StyleSheet } from 'react-native';
-import MapView from 'react-native-maps';
+import MapView, { type LatLng } from 'react-native-maps';
```

Comment:
I already fixed this once. This type does **not** exist. And it will be cast as an `any` type when you try to import non-existent types. And there is probably nothing worse than `any` in Flow... 🙂

Reply:
dammit 💩